lib/syscall_shim/uk_syscall.awk

```diff
@@ -56,4 +56,5 @@ END {
 	printf "\tva_end(ap);\n"
 	printf "\treturn ret;\n"
 	printf "}\n"
+	printf "long syscall() __attribute__ ((weak, alias (\"uk_syscall\")));\n"
```

Comment:
Is it an extra `)` at the end of the line?

Reply:
No?

Reply:
My bad, sorry for this.